Updated Swedish translations

Abbe98 opened the pull request:
Before merging this has to be tested(help wanted).

[Testing Translations](https://github.com/OpenOrienteering/mapper/wiki/Translation)

dg0yt commented on translations/OpenOrienteering_sv.ts:
```diff
@@ -438,12 +438,12 @@
         <location filename="../src/color_dock_widget.cpp" line="48"/>
         <source>CMYK</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
A finished entry must not be empty.

dg0yt replied:
I need to test if empty entries result in empty or untranslated ouput. However, it is at least difficult to tell whether the missing entry is intentionally or by mistake.

Abbe98 replied:
It's by mistake, updating it right away...
On Sep 21, 2015 7:44 AM, "Kai Pastor" <notifications@github.com> wrote:

> In translations/OpenOrienteering_sv.ts
> <https://github.com/OpenOrienteering/mapper/pull/518#discussion_r39942485>
> :
>
> > @@ -438,12 +438,12 @@
> >      <message>
> >          <location filename="../src/color_dock_widget.cpp" line="48"/>
> >          <source>CMYK</source>
> > -        <translation type="unfinished"></translation>
> > +        <translation></translation>
>
> I need to test if empty entries result in empty or untranslated ouput.
> However, it is at least difficult to tell whether the missing entry is
> intentionally or by mistake.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenOrienteering/mapper/pull/518/files#r39942485>.
>

Abbe98 replied:
Or looking into it it's not a mistake, sorry saw this from my phone.

The past translator did just leave the finished element empty when the
value should be equal to the source...

dg0yt commented on translations/OpenOrienteering_sv.ts:
```diff
@@ -438,12 +438,12 @@
         <location filename="../src/color_dock_widget.cpp" line="48"/>
         <source>RGB</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
A finished entry must not be empty.

dg0yt commented on translations/OpenOrienteering_sv.ts:
```diff
@@ -3824,22 +3824,22 @@ Skala om den importerade datan?</translation>
         <location filename="../src/map_editor.cpp" line="681"/>
         <source>&amp;PDF</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
A finished entry must not be empty.

dg0yt commented on translations/OpenOrienteering_sv.ts:
```diff
@@ -5114,7 +5118,7 @@ zooma med scrollhjulet om tillgängligt.</translation>
         <location filename="../src/util/item_delegates.cpp" line="154"/>
         <source>%</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
A finished entry must not be empty.

dg0yt commented on translations/OpenOrienteering_sv.ts:
```diff
@@ -5471,7 +5475,7 @@ zooma med scrollhjulet om tillgängligt.</translation>
         <location filename="../src/gui/print_widget.cpp" line="135"/>
         <source>mm</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
A finished entry must not be empty.

dg0yt commented on translations/OpenOrienteering_sv.ts:
```diff
@@ -5512,33 +5516,33 @@ zooma med scrollhjulet om tillgängligt.</translation>
         <location filename="../src/gui/print_widget.cpp" line="332"/>
         <source>PDF export</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
A finished entry must not be empty.

dg0yt commented on translations/OpenOrienteering_sv.ts:
```diff
@@ -5512,33 +5516,33 @@ zooma med scrollhjulet om tillgängligt.</translation>
         <location filename="../src/gui/print_widget.cpp" line="847"/>
         <source>dpi</source>
-        <translation type="unfinished">dpi</translation>
+        <translation></translation>
```
A finished entry must not be empty.

dg0yt commented on translations/OpenOrienteering_sv.ts:
```diff
@@ -5558,27 +5562,27 @@ zooma med scrollhjulet om tillgängligt.</translation>
         <location filename="../src/gui/print_widget.cpp" line="1005"/>
         <source>PNG</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
A finished entry must not be empty.

dg0yt commented:
Okay, than it should be okay. I will add your name to the list of contributors when I merge this change. Thank you very much.

dg0yt commented on translations/OpenOrienteering_sv.ts:
```diff
@@ -6236,68 +6240,68 @@ Högsta versionen som stöds är %2.</translation>
         <source>X:</source>
         <comment>x coordinate</comment>
-        <translation type="unfinished"></translation>
+        <translatio></translation>
```
This must be "translation", or Qt Linguist won't open the file.

dg0yt commented:
There are 685 translations for 1216 strings in total. Shall I wait for further updates before merging? Note that I will also have to run another round of updates to the untranslated strings soon.

Abbe98 commented:
Will do some more work in the coming days.

dg0yt commented:
Note: I updated all translations from the sources after this merge. So you will now find some new and changed source strings.

Abbe98 commented:
Woops, I have a translated most string but not yet committed, some
conflicts may appear now.
On Sep 27, 2015 9:44 PM, "Kai Pastor" <notifications@github.com> wrote:

> Note: I updated all translations from the sources after this merge. So you
> will now find some new and changed source strings.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenOrienteering/mapper/pull/518#issuecomment-143590125>
> .
>

dg0yt commented:
Don't mind, you can simply overwrite the file. Your version will have more translations, and the other parts get regenerated.

Abbe98 commented:
Great thanks then I make push before adding more strings.
On Sep 28, 2015 7:11 PM, "Kai Pastor" <notifications@github.com> wrote:

> Don't mind, you can simply overwrite the file. Your version will have more
> translations, and the other parts get regenerated.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenOrienteering/mapper/pull/518#issuecomment-143807142>
> .
>
